packages/toolkit/src/codewhisperer/commands/gumbyCommands.ts

```diff
@@ -0,0 +1,50 @@
+/*!
```

Comment:
should this file live in `packages/toolkit/src/amazonqGumby/` ?

Reply:
That should also work fine, and probably makes more sense since it will be away from `codewhisperer` files as we're technically a separate service

Reply:
that seems like it would be more intuitive

Reply:
this makes sense, I missed that